src/ButterCMS/ButterCMS.php

```diff
@@ -27,14 +27,18 @@ public function __construct($authToken)
         $this->client = new Client();
     }
 
-    protected function request($url, $params = [])
+    protected function request($url, $params = [], $tryCount = 0)
     {
         try {
             $params['auth_token'] = $this->authToken;
             $response = $this->client->get(self::API_ROOT_URL . $url, [
                 'query' => $params
             ]);
         } catch (ClientException $e) {
+            if ($tryCount < 1) {
+                return $this->request($url, $params, ++$tryCount);
+            }
+
             return false;
```

Comment:
The original implementation of this returned false, but throwing the exception itself is probably a better move. Since this is basically a breaking change anyways with the rename, I'll update this to throw the exception (or feel free to in the interim as I won't be able to until this evening).